simpleclient_eclipselink/src/main/java/io/prometheus/client/eclipselink/EclipseLinkStatisticsCollector.java

```diff
@@ -0,0 +1,744 @@
+                Object value = metrics.get(SessionProfiler.UowCommits);
+                if (value != null) {
+                  return ((Long) value).doubleValue();
+                }
+                return 0L;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_uow_created_total",
+            "Total number of created units of work",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.UowCreated);
+                if (value != null) {
+                  return ((Long) value).doubleValue();
+                }
+                return 0L;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_uow_released_total",
+            "Total number of released units of work",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.UowReleased);
+                if (value != null) {
+                  return ((Long) value).doubleValue();
+                }
+                return 0L;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_uow_rollbacks_total",
+            "Total number of unit of work rollbacks",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.UowRollbacks);
+                if (value != null) {
+                  return ((Integer) value).doubleValue();
+                }
+                return 0;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_cache_hits_total",
+            "Total number of cache hits",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.CacheHits);
+                if (value != null) {
+                  return ((Integer) value).doubleValue();
+                }
+                return 0;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_cache_misses_total",
+            "Total number of cache misses",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.CacheMisses);
+                if (value != null) {
+                  return ((Integer) value).doubleValue();
+                }
+                return 0;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_change_sets_processed_total",
+            "Total number of processed change sets",
```

Comment:
Are this one and the next one successes and failures? Failures and total is a bit easier to work with